ecl/provider_connectivity/v2/tenant_connections/testing/fixtures.go

```diff
@@ -0,0 +1,639 @@
+        "name": "test_name_1",
+        "description": "test_desc_1",
+		"tags": {
+			"test_tags1": "test1"
+		},
+		"tenant_connection_request_id": "21b344d8-be11-11e7-bf3c-0050569c850d",
+		"device_type": "ECL::Compute::Server",
+		"device_id": "8c235a3b-8dee-41a1-b81a-64e06edc0986",
+		"attachment_opts": {
+			"fixed_ips": [
+				{
+					"ip_address": "192.168.1.1",
+					"subnet_id": "1f424165-2202-4022-ad70-0fa6f9ec99e1"
+				}
+			],
+			"allowed_address_pairs": [
+				{
+					"ip_address": "192.168.1.2",
+					"mac_address": "11:22:33:aa:bb:cc"
+				}
+			]
+		}
+    }
+}
+`
+
+// CreateAttachComputeServerResponse provides the output from a Create request.
+const CreateAttachComputeServerResponse = `
+{
+    "tenant_connection":{
+        "id": "2a23e5a6-bd34-11e7-afb6-0050569c850d",
+        "tenant_connection_request_id": "5fbcc350-bd33-11e7-afb6-0050569c850d",
+        "name": "test_name_1",
+        "description": "test_desc_1",
+        "tags": {
+			"test_tags1": "test1"
+		},
+        "tenant_id": "7e91b19b9baa423793ee74a8e1ff2be1",
+        "name_other": "",
+        "description_other": "",
+        "tags_other": {},
+        "tenant_id_other": "c7f3a68a73e845d4ba6a42fb80fce03f",
+        "network_id": "77cfc6b0-d032-4e5a-b6fb-4cce2537f4d1",
+        "device_type": "ECL::Compute::Server",
+        "device_id": "8c235a3b-8dee-41a1-b81a-64e06edc0986",
+        "device_interface_id": "",
+        "port_id": "b404ed73-9438-41a1-91ed-49d0e403be64",
+        "status": "creating"
+    }
+}
+`
+
+// CreateAttachBaremetalServerRequest provides the input to a Create request.
+const CreateAttachBaremetalServerRequest = `
+{
+    "tenant_connection": {
+        "name": "attach_bare_name",
+        "description": "attach_bare_desc",
+		"tags": {
+			"test_tags1": "test1"
+		},
+		"tenant_connection_request_id": "147c4ffa-481e-11ea-8088-525400060300",
+		"device_type": "ECL::Baremetal::Server",
+		"device_interface_id": "46eb7624-d462-46c2-8ac7-f988a15d3280",
+		"device_id": "0acab22f-8993-451c-8a6b-398b0244f578",
+		"attachment_opts": {
+			"segmentation_id": 10,
+			"segmentation_type": "flat",
+			"fixed_ips": [
+				{
+					"ip_address": "192.168.1.1",
+					"subnet_id": "1f424165-2202-4022-ad70-0fa6f9ec99e1"
+				}
+			],
+			"allowed_address_pairs": [
+				{
+					"ip_address": "192.168.1.2",
+					"mac_address": "11:22:33:aa:bb:cc"
+				}
+			]
+		}
+    }
+}
+`
+
+// CreateAttachBaremetalServerResponse provides the output from a Create request.
+const CreateAttachBaremetalServerResponse = `
+{
+    "tenant_connection":{
+        "id": "0d956a2e-4958-11ea-8088-525400060300",
+        "tenant_connection_request_id": "147c4ffa-481e-11ea-8088-525400060300",
+        "name": "attach_bare_name",
+        "description": "attach_bare_desc",
+        "tags": {
+			"test_tags1": "test1"
+		},
+        "tenant_id": "7e91b19b9baa423793ee74a8e1ff2be1",
+        "name_other": "",
+        "description_other": "",
+        "tags_other": {},
```

Comment:
??
```suggestion
        "tags_other": "",
```

Reply:
tags_other is not a string, so I guess it works fine with {}.